docs/features/techdocs/FAQ.md

```diff
@@ -29,9 +29,9 @@ plugins (e.g.
 [MkDocs Monorepo Plugin](https://github.com/spotify/mkdocs-monorepo-plugin)) as
 well as a selection of Python Markdown extensions that TechDocs supports.
 
-#### Does TechDocs support file formats other than Markdown (e.g. rst, asciidoc) ?
+#### Does TechDocs support file formats other than Markdown (e.g. RST, AsciiDoc) ?
 
 Not right now. We are currently using MkDocs to generate the documentation from
-source. So, they have to be in Markdown format. However, in future we want to
-support other alternatives to MkDocs. That will make it possible to use other
+source, so the files have to be in Markdown format. However, in the future we want to
+support other static site generators which will make it possible to use other
 file formats.
```

Comment:
```suggestion
source, so the files have to be in Markdown format. However, in the future we
want to support other static site generators which will make it possible to use
otherfile formats.
```

Reply:
Oops! Just noticed here that "other file" should be two separate words. This is how it's being rendered at https://backstage.io/docs/features/techdocs/faqs:

![techdocs](https://user-images.githubusercontent.com/51410533/94802028-39623c00-03e7-11eb-9721-0f6850030aae.png)